AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -161,6 +162,47 @@ extension CardViewController {
             return holderNameItem
         }()
 
+        internal lazy var coBadgedCardItem: FormCoBadgedCardItem = {
+            let item = FormCoBadgedCardItem(
+                title: localizedString(
+                    .creditCardDualBrandTitle,
+                    configuration.localizationParameters
+                ),
+                subtitle: localizedString(
+                    .creditCardDualBrandDescription,
+                    configuration.localizationParameters
+                ),
+                selectableFormItems: [],
+                style: .init(
+                    title: configuration.style.sectionHeader,
+                    subtitle: configuration.style.footnoteLabel
+                )
+            )
+            item.style.subtitle.textAlignment = .left
```

Comment:
Have you tested this with RTL languages? I feel like we shouldn't need to specify this manually

Reply:
The section header is centered that's why the manual setting - I guess `.natural` would be the one to choose here